tests/user_restricted/data/pds_scenarios.py

```diff
@@ -1,42 +0,0 @@
-from ..configuration.config import TEST_PATIENT_ID, SPINE_HOSTNAME
```

Comment:
so I guess this is converted to bdd in the feature files, can the data dir be deleted?

Reply:
As above.